cleanlab/count.py

```diff
@@ -1176,8 +1178,10 @@ class 0, 1, ..., K-1. `pred_probs` should have been computed using 3 (or
     """
 
     if multi_label:
-        # [0, 1, 2, ... num_classes - 1]
-        unique_classes = range(pred_probs.shape[1])  # Assumes all classes are represented in labels
+        # Assumes all classes are represented in labels: [0, 1, 2, ... num_classes - 1]
+        unique_classes = range(
```

Comment:
I think `unique_classes` can be defined before the if/else block, I scrolled down the code a couple lines and right now `pred_probs.shape[1]` is still being called in the else block (should also be changed to `get_num_classes()` but probably was just missed out in your previous commit), but since they both use `unique_classes`, moving it out would be less code duplication?

Reply:
good catch!